client/py/synnax/framer/streamer.py

```diff
@@ -39,47 +38,99 @@ class _Response(Payload):
-        from_: CrudeTimeStamp | None = None,
     ) -> None:
-        self.from_ = from_ or TimeStamp.now()
-        self.__stream = client.stream(_ENDPOINT, _Request, _Response)
-        self.__adapter = adapter
+        self._stream = client.stream(_ENDPOINT, _Request, _Response)
+        self._adapter = adapter
         self.__open()
 
     def __open(self):
-        self.__stream.send(_Request(keys=self.__adapter.keys, start=self.from_))
+        self._stream.send(_Request(keys=self._adapter.keys))
 
     @overload
-    def read(self, timeout: float) -> Frame | None:
+    def read(self, timeout: float | int | TimeSpan) -> Frame | None:
+        """Reads the next frame of telemetry from the streamer with a timeout. If no
+        frame is received within the timeout, this method will return None. If an error
+        occurs while reading the frame, an exception will be raised.
+
+        :param timeout: The maximum amount of time to wait for a frame to be received
+        before returning None. This can be a float or integer representing the number
+        of seconds, or a synnax TimeSpan object.
+        :return: The next frame of telemetry, or None if no frame is received within the
+        timeout.
+        """
         ...
 
     @overload
     def read(self) -> Frame:
+        """Reads the next frame of telemetry from the streamer, blocking until a frame
+        is received. If an error occurs while reading the frame, an exception will be
+        raised.
+        """
         ...
 
     def read(self, timeout: float | None = None) -> Frame | None:
+        """Reads the next frame of telemetry from the streamer. If a timeout is provided,
+        this method will wait for the specified amount of time for a frame to be received.
+        If no frame is received within the timeout, this method will return None. If no
+        timeout is provided, this method will block until a frame is received.
+
+        If an error occurs while reading the frame, an exception will be raised.
+
+        :param timeout: The maximum amount of time to wait for a frame to be received
+        before returning None. If no timeout is provided, this method will block until a
+        frame is received.
+        :return: The next frame of telemetry, or None if no frame is received within the
+        timeout.
+        """
         try:
-            res, err = self.__stream.receive(timeout)
-            if err is not None:
-                raise err
-            return self.__adapter.adapt(Frame(res.frame))
+            res, exc = self._stream.receive(TimeSpan.to_seconds(timeout))
+            if exc is not None:
+                raise exc
+            return self._adapter.adapt(Frame(res.frame))
         except TimeoutError:
             return None
 
-    def close(self, timeout: float | None = None):
-        exc = self.__stream.close_send()
+    def update_channels(self, channels: ChannelParams):
+        """Updates the list of channels to stream.
```

Comment:
Clarify that this replaces the current lists of channels with the new list, and doesn't add to them.